### ISSUE TITLE
Flickering in Safari 

### ISSUE THREAD

**Issue by ertdfgcvb** [nolimits4web/swiper#3527]
This is a (multiple allowed):
* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)

* Swiper Version: Swiper 5.3.6
* Platform/Target and Browser Versions: macOS 10.14.6, Safari 13.1

### What you did
In safari there is a brief flicker when the swipe animation ends. This can result in a very annoying experience. I noticed that there was a similar issue (#3124) flagged as "outdated".
It seems hardware related. 

### Workaround 
Added `-webkit-transform: translate3d(0, 0, 0);` to the `.swiper-slide` class as suggested [here](https://stackoverflow.com/questions/15751012/css-transform-causes-flicker-in-safari-but-only-when-the-browser-is-2000px-w#15759785). 
The `-webkit` prefix guarantees that the fix is effective only on Safari.
Strangely it seems that the added 3d transform on this class _disables_ the hardware acceleration as the transition appears less smooth (might be an impression).

**Comment by nortonandreev** [comment]
Have fixed it by adding:
    -webkit-transform: translateZ(0);
    -webkit-backface-visibility: hidden;

However, yes, I think it would be good if it gets fixed.

**Comment by ertdfgcvb** [comment]
I captured the scroll from one frame to the other with a screen recorder the moment the transition is ending. 
In the second image the 'flickering' area is visible… It's just about one frame long.
This seems related to a Safari issue dealing with the hardware renderer… so it seems that the only option is to trigger software rendering with the CSS fixes mentioned above.
![swiper_safari_bug_1](https://user-images.githubusercontent.com/864073/78470594-1baf6500-772b-11ea-8b6e-f0dd6ca66f09.png)
![swiper_safari_bug_2](https://user-images.githubusercontent.com/864073/78470597-210caf80-772b-11ea-8720-a5ff31dd654f.png)

**Comment by tiadaguru** [comment]
This bug is reproducible on **_every Swiper installation_**, including the latest Swiper 5.3.7, on iOS Safari 13.4.1.

It even reproduces on [**swiperjs.com**](https://swiperjs.com/demos/). It's been mentioned both in that recent issue and I believe [this StackOverflow post](https://stackoverflow.com/questions/32639639/ios-9-mobile-safari-has-a-blinking-bug-with-transform-scale3d-and-translate3d), too. [**A short screen recording of the bug on Swiperjs.com on April 19, 2020**](https://streamable.com/7nqy71)

Test device: iPhone 6S Plus, iOS 13.4.1

I spent 45 minutes tracking down this bug. :( Thank you for offering a workaround, but either the root cause fix or this workaround should ideally be pushed to a public release. I can't imagine we are the only ones who've noticed this.

**Comment by tiadaguru** [comment]
After some testing, this workaround (`-webkit-transform: translateZ(0)`) tanks swiping performance, as expected (I'd wager the performance degradation is more noticeable as you move beyond simple sliders). The default 3dtranslate offers smooth stutter-free transitions, but with flickering. 

Alas.

**Comment by tiadaguru** [comment]
I took the video frame by frame. It seems like, when it flickers once on a certain slide, it will flicker _**again**_ exactly at that slide location. 

# Some examples

Format of `Seconds.FrameNumber`. Thus, at 7.02 , 9.07, 12.03, and 13.11, Swiper.js has sliced off the **_same_** slide at the **_same_** location, i.e,. the orange sunset around halfway through its reflection. Three times, same slice: sometimes, Swiper.js keeps the left and sometimes, Swiper.js keeps the right.

## 7.02 / 9.07 / 12.03 / 13.11 set

### 7 sec, 02 frame (see the slice location)
![7nqy71 00_00_07_02 Still001](https://user-images.githubusercontent.com/56522083/80265905-39575680-8667-11ea-9028-5969e3d0a55c.png)

### 12 sec, 03 frame (same as above!)
![7nqy71 00_00_12_03 Still003](https://user-images.githubusercontent.com/56522083/80265936-5c820600-8667-11ea-92bf-b8e4ca059d67.png)

### 9 sec, 7 frame (again, same)
![7nqy71 00_00_09_07 Still002](https://user-images.githubusercontent.com/56522083/80265927-54c26180-8667-11ea-842c-9429c16996de.png)

### 13 sec, 11 frame (again, same)
![7nqy71 00_00_13_11 Still004](https://user-images.githubusercontent.com/56522083/80265937-5ee46000-8667-11ea-801f-43b7ff69fc68.png)

## 21.21 / 29.04 / 30.04 / 31.04 set

### 21 sec, 21 frame
![7nqy71 00_00_21_21 Still007](https://user-images.githubusercontent.com/56522083/80265944-69065e80-8667-11ea-9746-07b429e81b84.png)

### 29 sec, 04 frame
![7nqy71 00_00_29_04 Still008](https://user-images.githubusercontent.com/56522083/80265995-a965dc80-8667-11ea-8ea4-9a76c77dafd8.png)

### 30 sec, 04 frame
![7nqy71 00_00_30_04 Still009](https://user-images.githubusercontent.com/56522083/80265997-acf96380-8667-11ea-82a7-17b4c7d1b24e.png)

### 31 sec, 04 frame
![7nqy71 00_00_31_04 Still010](https://user-images.githubusercontent.com/56522083/80265998-aec32700-8667-11ea-9798-035f350649c4.png)

There are more examples, but this post is probably illustrative enough.

**Comment by tiadaguru** [comment]
I can confirm the bug remains in the latest 5.3.8 update released today and still reproduces perfectly on swiperjs.com. Tomorrow, I'll see what Safari's debugging can share.

I may have misunderstood the hardware inclination, but I don't think this is hardware-specific. I can reproduce it reliably in iOS 13.4.1 (iPhone 6S Plus), while @ertdfgcvb can reproduce it on macOS 10.14.6 (presumably a laptop/desktop).

**Comment by ertdfgcvb** [comment]
I think this is not a swiper issue: I'm able to reproduce this flickering on Safari (mobile and desktop) just with simple horizontal scrolling (meanwhile I wrote my own "swiper", partly to see if I was able to overcome this issue)… 
I guess it has to do with the hardware accelerated rendering; the only way to avoid it is with the CSS triggers/hacks above; but this will kill the fps for a smooth transition. 
The fact that it appears in the same spot is probably related to the texture sizes the browser uses to render the page (this is speculation, of course).
In any case: just try to build a very long horizontal page and scroll along… you'll notice the same flickering.

**Comment by tiadaguru** [comment]
Interesting. On every horizontal scrolling site on my iOS device, there are _**zero**_ flickers. 

A few horizontal scrolling test sites; to confirm, you're seeing flicker on _**all**_ of these?

1. Sway (using their swiping JS library): https://sway.office.com/howtosway (hit Settings to switch to horizontal mode)
2. CSS Tricks CSS Horizontal Scrolling Demo: https://css-tricks.com/pure-css-horizontal-scrolling/ (scroll down to the Codepen demo)
3. Webflow Horizontal Scrolling Template Demo: http://horizontal-scrolling-sticky-section.webflow.io/ (not great UI, but still another system)

I saw no flicker on the above sites whatsoever. SwiperJS remains flickery: I can even trigger it on the blank slide demos, though it's much worse on image-based slides.

**Comment by roman-rr** [comment]
Related issue on webkit community 
https://bugs.webkit.org/show_bug.cgi?id=212130

**Comment by vercordio** [comment]
`cssMode: true` fixed the flicker for me, without the performance penalty from hardware acceleration - but of course it reduces Swiper functionality considerably.

**Comment by ieozt38** [comment]
> `cssMode: true` fixed the flicker for me without the performance penalty of hardware acceleration hacks (`translate3d`, `backface-visibility`) but of course it reduces Swiper functionality considerably.
> 
> Likely only appropriate for simpler Swiper implementations.

where to put cssMode:true ??

**Comment by vercordio** [comment]
> where to put cssMode:true ??

@ieozt38 add this as a parameter, like this:

`var mySwiper = new Swiper('.swiper-container', {
    cssMode: true
});`

More info [in the docs](https://swiperjs.com/api/#parameters), under "CSS Scroll Snap".

**Comment by ieozt38** [comment]
> > where to put cssMode:true ??
> 
> @ieozt38 add this as a parameter, like this:
> 
> `var mySwiper = new Swiper('.swiper-container', { cssMode: true });`
> 
> More info [in the docs](https://swiperjs.com/api/#parameters), under "CSS Scroll Snap".

Thank you very much but sadly the behaviour changes too much with that property :(

**Comment by tiadaguru** [comment]
I've done some limited testing and this issue looks to be fixed for me on Swiper 6.1.3 and iOS 13.6.1. I'm using an iPhone 6s Plus on the default Safari browser.

I can no longer reproduce it on https://swiperjs.com/demos/ nor on my own internal sites. I can swipe slowly, quickly, etc.: no flicker, no blackouts, no jittery splicing. Looks as smooth as can be and is a far more reliable & consistent experience.

Notably, I am _not_ using CSS mode: just the default JS-based animations.

**Comment by ieozt38** [comment]
> I've done some limited testing and this issue looks to be fixed for me on Swiper 6.1.3 and iOS 13.6.1. I'm using an iPhone 6s Plus on the default Safari browser.
> 
> I can no longer reproduce it on https://swiperjs.com/demos/ nor on my own internal sites. I can swipe slowly, quickly, etc.: no flicker, no blackouts, no jittery splicing. Looks as smooth as can be and is a far more reliable & consistent experience.
> 
> Notably, I am _not_ using CSS mode: just the default JS-based animations.

Do you mean 6.1.2 because i don't see release version 6.1.3. Anyway thanks

**Comment by Guyr989** [comment]
> Have fixed it by adding:
> -webkit-transform: translateZ(0);
> -webkit-backface-visibility: hidden;
> 
> However, yes, I think it would be good if it gets fixed.

adding where?

**Comment by codevon** [comment]
I use custom onPrev & onNext instead of Swiper:navigation to solved this issue.
I don't why, but it works well for me, :)

`
    onPrev () {
      this.swiper.slidePrev();
    },
    onNext () {
      this.swiper.slideNext();
    }
`

**Comment by 91DarioDev** [comment]
I started using swiper a couple of days ago and I noticed the bug very soon. 

I'm on swiper 6.3.4￼ and iOS 14.0.1 and iPhone XS

**Comment by anthonyhenson** [comment]
I use Swiper from CDN. I experience the image flicker when panning a zoomed image on iOS and Android web browsers (for Android, this happened much less on the Chrome browser than the default Samsung browser). Looks to me like it's doing a re-draw every time you pan the image. Maybe more of an issue with high res images?

**Comment by NeonilaIvanova** [comment]
> it is still present in Safari in macOS Big Sur

I also encountered this issue on Safari macOS Big Sur.

**Comment by moritzebeling** [comment]
I experience the problem consistently over the last years, with every new version of macOS, Safari, Swiper.
`-webkit-transform: translate3d(0, 0, 0);` helps a little bit, but doesn’t solve it completely...

**Comment by vltansky** [comment]
#3641 may be related.
Can somebody confirm that @zhengsusu2014 solution works?
```css
.swiper-slide{
-webkit-backface-visibility: hidden;
-webkit-transform: translate3d(0,0,0);
}

.swiper-wrapper{
-webkit-transform-style: preserve-3d;
}
```

**Comment by vltansky** [comment]
@tiadaguru @ieozt38 @Guyr989 I see that it was working properly for you in `6.1.2` release according to your messages.
Although some users here said that the bug still persists, could you please take a look if it still reproduces for you in the latest version?
Unfortunately, I don't have iPhone / mac :)

**Comment by MentalGear** [comment]
Same issue, adding on macOS Safari. Seems to be fixed by

> Have fixed it by adding:
> -webkit-transform: translateZ(0);
> -webkit-backface-visibility: hidden;
> 
> However, yes, I think it would be good if it gets fixed.

**Comment by Yankai17148** [comment]
same flicker problem on desktop Chrome 87.0.4280, maybe #3641 can fix this problem.

**Comment by Franz-Kafka** [comment]
This problem is still there? I have same issue here on Safari… 
All the workarounds are not working for me. So there is no solution for this issue?

**Comment by WesWedding** [comment]
Not a universal one, no.  It seems likely that any existing solutions suggested so far will only fix specific types of situations and that unique use-cases and browser combinations won't be addressed by a single solution.  Try to experiment with applying the tricks to different parts of your DOM and see what happens.

**Comment by tiadaguru** [comment]
> @tiadaguru @ieozt38 @Guyr989 I see that it was working properly for you in `6.1.2` release according to your messages.
> Although some users here said that the bug still persists, could you please take a look if it still reproduces for you in the latest version?
> Unfortunately, I don't have iPhone / mac :)

My apologies for the long delay. Unfortunately, I no longer have the iPhone 6S Plus. 

I've re-tested on an iPhone 12 Pro on iOS 14.6 in Safari, with the swiperjs.com/demos build (Thumbs Gallery again; I presume 6.6.2) and the bug does not reproduce there.

Would anyone with an iPhone 6S be able to check? I

**Comment by moritzebeling** [comment]
@vltansky I can confirm that it improves the flickering situation. But wouldn’t say that it solves it completely. Without, I have the flickering with every slide, using this hack maybe only every 5th or 10th slide.
```css
.swipe-slide {
  -webkit-backface-visibility: hidden;
  -webkit-transform: translate3d(0,0,0);
}
.swiper-wrapper {
 -webkit-transform-style: preserve-3d;
}
```

**Comment by adambichler** [comment]
As of today, the problem was still present in version 6.7.1.
For me, adding this to the .swiper-slide helped for em
`
-webkit-transform: translate3d(0, 0, 0);
`

**Comment by vltansky** [comment]
@moritzebeling @adambichler try `cssMode` https://swiperjs.com/swiper-api

**Comment by mitschmidt** [comment]
Confirming this bug using the latest swiper version (6.8.1) on an iPhone 7 with iOS 14. cssMode solves the flickering, but we need the zoom functionality for our app. Will try to find a different solution.

**Comment by wwwisie** [comment]
> Confirming this bug using the latest swiper version (6.8.1) on an iPhone 7 with iOS 14. cssMode solves the flickering, but we need the zoom functionality for our app. Will try to find a different solution.
> 
> EDIT: I must have messed up the various solutions; After only adding `-webkit-transform: translate3d(0, 0, 0);` to the `.swiper-slide` class it works without flickering on the iPhone 7. Thanks to everybody for providing their ideas and solutions, let's hope the root cause can be solved by Safari or the swiperjs team.

This worked for me, even on older versions. I've tested in v4.3.3 and v6.0.4.

**Comment by AdrianFahrbach** [comment]
I was redirected to this issue from #5180 but I don't think that those problems are actually duplicates.

The suggested solutions from this thread don't work for me.
The flickering that I described only happens while sliding and not on the transition end. I addition to that it doesn't flicker on desktop Safari. For further details take a look at #5180

**Comment by DanielOberlechner** [comment]
Hey I'have the same issue on macos and safari.
I have 3 slides and as effect cube.
When the 3. slide is done, it goes once again to show the 3. one which starts to flicker as hell.
Then the 3. is done the second time it starts at the beginning.

Can confirm that this flickering thing is still a issue.

Anyone any ideas how to fix that problem?
I'm not long into this library simply started fresh and have this issue.

Kind regards,
Daniel Oberlechner

**Comment by stefanovacchi** [comment]
> @vltansky I can confirm that it improves the flickering situation. But wouldn’t say that it solves it completely. Without, I have the flickering with every slide, using this hack maybe only every 5th or 10th slide.
> 
> ```css
> .swipe-slide {
>   -webkit-backface-visibility: hidden;
>   -webkit-transform: translate3d(0,0,0);
> }
> .swiper-wrapper {
>  -webkit-transform-style: preserve-3d;
> }
> ```

another improvement in besides this is to disable followFinger.

**Comment by maenamer** [comment]
below worked perfectly for me when using Safari. I hope the same should work with you to stop Flickering in Safari .

`.swiper-slide {
-webkit-transform: translateZ(0);
-o-transform: translateZ(0);
-moz-transform: translateZ(0);
-ms-transform: translateZ(0);
-webkit-backface-visibility:hidden;
transform: translateZ(0); }
.swiper-wrapper {
 -webkit-backface-visibility:hidden;
 -webkit-transform-style: preserve-3d;
}`

**Comment by vltansky** [comment]
looks like adding solving the issue
```css
.swiper-slide{
   trasnform: translateZ(0);
}
```

@nolimits4web what do you think?
we can add some hacks for safari only for example:
```
@media screen and (min-color-index:0) and(-webkit-min-device-pixel-ratio:0)
{ @media {
    /*
      Define here the CSS styles applied only to Safari browsers
      (any version and any device)
     */
}}
```
https://solvit.io/bcf61b6
https://stackoverflow.com/questions/16348489/is-there-a-css-hack-for-safari-only-not-chrome

**Comment by moritzebeling** [comment]
Latest Hardware, latest Version of Safari and Swiper, all the CSS hacks discussed above in place, the flickering remains. :(

When I use `cssMode`, the transitions are not as smooth and the loop interrupts once I have clicked through all images.

Will this ever be fixed? I do see other sliders on the web that don’t have flickering... Is it time to look for another solution?

**Comment by wplit** [comment]
> Latest Hardware, latest Version of Safari and Swiper, all the CSS hacks discussed above in place, the flickering remains. When I use the autoplay module, the flickering appears with every slide, otherwise it’s only every nth :(
> 
> When I use `cssMode`, the transitions are not as smooth and the loop interrupts once I have clicked through all images.
> 
> Will this ever be fixed? I do see other sliders on the web that don’t have flickering... Is it time to look for another solution?

Do you have an example URL where you can still see flickering after adding the translateZ(0); solution on the slides? I'm testing it now across lots of devices and it solves it for me. Interested to see where it fails.

**Comment by bikubi** [comment]
> After some testing, this workaround (`-webkit-transform: translateZ(0)`) tanks swiping performance, as expected (I'd wager the performance degradation is more noticeable as you move beyond simple sliders).

> cssMode: true fixed the flicker for me without the performance penalty of hardware acceleration hacks

Chiming in re: performance penalty, which in my case (75 slides with jpegs) was so bad that it either crashed the browser (mobile Safari on iPad, iOS 15.1) or, with slide count halved, effectively prevented any animation.  
After I removed the hacks everything went back to smooth, the original issue got better, I think (in the meantime I had "solidified" the base layout). `cssMode` would have worked as well, but introduced a different flicker which I did not attempt to debug.

**Comment by wplit** [comment]
Just adding some extra info here that I found while troubleshooting.. Not solutions, but gives some insight into the problem.

- Removing the overflow: hidden from the 'swiper' class completely removes the flickering (obviously not useful as we need it to be hidden, but could give a clue to the best solution - eg can we recreate overflow hidden, but avoid using overflow?)
- The flickering is occuring on swiper-wrapper, not the individual slides. (you can test by adding background colours to all the levels, if the slides were flicking you'd see the wrapper when it flickers, but you don't)
- Adding translateZ(0) on the slides solves it, but with a large number of slides causes performance issues because you're creating too many layers for it to handle.

**Comment by vltansky** [comment]
@wplit 
https://codepen.io/vltansky/pen/rNYVBJa
take a look, how is it works for you?

**Comment by DanielOberlechner** [comment]
@vltansky 
At my 16 Inch MacBook Pro with Apple M1 Max everything run's fine without any flickering.
This special slide seems to work flawlessly ....

Kind regards,
Daniel

**Comment by wplit** [comment]
@vitansky

Still seeing performance issue, but I don't think it can be avoided. Obv 20 full screen slides is an extreme scenario, so this solution will solve it for 95-99% of use cases i would have thought.

**Comment by vltansky** [comment]
Swiper v8 released with new option https://swiperjs.com/swiper-api#param-maxBackfaceHiddenSlides
give it a try

**Comment by moritzebeling** [comment]
I now have the impression that this reduces the flickering a little bit (only every 3rd-5th instead of every slide):
```css
.swipe-slide {
  -webkit-backface-visibility: hidden;
  -webkit-transform: translate3d(0,0,0);
}
.swiper-wrapper {
 -webkit-transform-style: preserve-3d;
}
```
while the following reduces the flickering even more (maybe every 10-20th):
```css
.swiper-slide {
    -webkit-backface-visibility: hidden;
    -webkit-transform: translateZ(0);
}
.swiper-wrapper {
    -webkit-transform-style: preserve-3d;
    -webkit-transform: translateZ(0);
}
```

**Comment by DeltaRays** [comment]
I tried using all the suggested changes however nothing fixed it. It works fine on desktop, even with small viewports. I've removed as much other code as possible to see if it was caused by something I'd done but nothing seems to have improved. Dragging the slide to switch to the next one isn't an issue, however when I stop dragging on it (aka stop touching the screen with my finger) it doesn't transition to the next slide, instead it waits some time then replaces the current one with the next one...

**Comment by enriquejr99** [comment]
One solution is to apply `-webkit-transform: translateZ(0);` on the hidden slides that are about to become visible (instead of all slides) and removing the property once the animation is done. This fixes the flickering and shouldn't cause performance issues as `translateZ(0)` is only applied to 1 or 2 slides in most cases.

However, when the user navigates to a faraway slide (e.g. from the last slide back to the first slide) this solution could cause performance issues. In this case, the `transition-timing-function` should be considered. For example, for `transition-timing-function: ease-in-out;` you could apply `translateZ(0)` to the first 2 and the last 2 slides, as these will move the slowest and hence is where the flickering will be most noticeable.

**Comment by marianobenitezz** [comment]
Still not working properly. I'm using Swiperjs 8.1 and iOS 15. When the swiper moves the transition is not smooth. You can move it but when you release the finger it lags for a second and then it pass to the next slide without continuing the animation. None of the solutions above worked.

Here's an example video, in the first ones it was me moving the slider, then, the autoplay

https://user-images.githubusercontent.com/70652590/178338820-dd916a13-a3d9-49f7-9923-a7ebf14a5675.mp4

**Comment by HongsCode** [comment]
I tried two solution. but issue don't resolve perfectly.
- [ ] adding,  -webkit-transform:traslateZ(0), use 'cssMode={true}'
       -> last slide appear an empty slide....so I tried use 'transition-timing-function ...so don't resolve

- [ ] update v8 and use  maxBackfaceHiddenSlides
      -> There are still an issue.

Does this problem only come from safari?
Can you explain what caused it is?

**Comment by DanielOberlechner** [comment]
@HongsCode Did you try already to upgrade to the newest version of the slider? Came out a short time ago...

**Comment by HongsCode** [comment]
@DanielOberlechner Yes, I tried v8.3.1 but issue don't resolve.

Inside the swiper, there is a lotties player, and the swiper has the autoplay property.
I think swiper rendering and lotties player rendering clash at same time.

**Comment by HongsCode** [comment]
in my case...
apply to translate3d or tranlateZ then issue resolve 
but a blank page is displayed on the last page, and the swipergesture does not work.

**Comment by HongsCode** [comment]
It was reolved by the code below

.swipe-slide {
  -webkit-backface-visibility: hidden;
  -webkit-transform: translate3d(0,0,0);
}
.swiper-wrapper {
 -webkit-transform-style: preserve-3d;
}

thank you

**Comment by DanielOberlechner** [comment]
I only had some issues if I was used as using a cube and or similar effects.

**Comment by jcsdesterke** [comment]
I am having this issue with version 7.3.2, does anyone have a fix?
I have tried the webkit transform solutions (translate3d, translateZ etc.) but that doesn't seem to work.

**Comment by vxm5091** [comment]
TLDR: adding these styles to the components inside each slide that appeared to flicker fixed the issue for me. 
```
backdrop-filter: blur(3.5px);
-webkit-backdrop-filter: blur(3.5px);
-moz-backdrop-filter: blur(3.5px);
```
Note: the backdrop-filter effect/value can be anything, that's just what I used.

Not sure if my issue is more specific but in case this helps someone - I nailed down the cause of the flickering in my case to a clash between Gatsby `StaticImage` / `GatsbyImage` and elements that were `absolute` positioned on top. Those elements would get overlaid by the image from `touchEnd` up until the slide animation completed. Hence the flickering. 

iOS Safari 15.6.1 (was also happening in Firefox and Brave on iPhone)
Swiper 8.3.2

**Comment by liamcrean** [comment]
> ```
> backdrop-filter: blur(3.5px);
> -webkit-backdrop-filter: blur(3.5px);
> -moz-backdrop-filter: blur(3.5px);
> ```

This worked for me - with absolutely positioned images inside slides with `object fit:cover;`

Thanks

**Comment by amerimnos** [comment]
After 20 hours of trying in all the ways listed above, the conclusion.
Adding -webkit-transform:translate3d (0, 0, 0) to .swiper-slide eliminates flickering during continuous swipe.
After removing the cache, the phenomenon of flickering 1 time when swiping after the first load is resolved by removing overflow:hidden from the parent tag.

( 위에 열거한 모든 방법으로 20시간 동안 시도한 끝에 결론.
.swiper-slide에 -webkit-transform: translate3d(0, 0, 0) 추가하면 연속적으로 스와이프 할 때 깜박이는 현상 제거됨.
캐시 제거 후 첫 로딩 후 스와이프 할 때 1번 깜박이는 현상은 부모 태그에 overflow:hidden 제거 하면 해결 됨. )

**Comment by rczestgeek** [comment]
Sorry to say that the above-given methods are not working for me. If anyone has any new ideas to fix this Flickering issue please share with me. I will be very thankful to you

**Comment by jansengunderson** [comment]
> 

Was having the same issue with Gatsby Image. Applying backdrop-filter did not work for me.
However, adding:
`transform: translateZ(1px); `
to the absolutely positioned element solved the Safari problem.

**Comment by mustafa-soylemez** [comment]
@adambichler forcing `translateZ(0)` to `swiper-slide` class with important broke whole swiper

**Comment by nolimits4web** [comment]
Issue is closed because of outdated/irrelevant/not actual/needed more information/inactivity.

If this issue is still actual and reproducible for latest version of Swiper, please create new issue and fill the issue template correctly:
* Clearly describe the issue including steps to reproduce when it is a bug.
* Make sure you fill in the earliest version that you know has the issue.
* Provide live link or JSFiddle/Codepen or website with issue
